faculty_mill/publish.py

```diff
@@ -0,0 +1,94 @@
+import os
+from pathlib import Path
+from typing import Optional
+from uuid import UUID
+from time import sleep
+import click
+import sherlockml
+import sml.auth
```

Comment:
`sml.auth.user_id()` should be replaced with the `AccountClient` in `sherlockml`. In general, we should avoid importing `sml` in other libraries.